modules/comment-likes/comment-like-count.js

```diff
@@ -0,0 +1,55 @@
+jQuery( document ).ready( function( $ ) {
+	var jsonAPIbase = 'https://public-api.wordpress.com/rest/v1',
+		APIqueue = [];
+
+	function getCommentLikeCounts() {
+		$( '.comment-like-count' ).each( function() {
+			var blogId = $( this ).attr( 'data-blog-id' ),
+				commentId = $( this ).attr( 'data-comment-id' );
+
+			APIqueue.push( '/sites/' + blogId + '/comments/' + commentId + '/likes' );
+		} );
+
+		fetchCounts();
+	}
+
+	function showCount( commentId, count ) {
+		if ( count < 1 ) {
+			return;
+		}
+
+		$( '#comment-like-count-' + commentId ).find( '.like-count' ).hide().text( count ).fadeIn();
+	}
+
+	function fetchCounts() {
+		var batchRequest = {
+			path: '/batch',
+			data: 'urls[]=' + APIqueue.join( '&urls[]=' ),
+			success: function( response ) {
+				for ( var path in response ) {
+					if ( ! response[ path ].error_data ) {
+						var urlPieces = path.split( '/' ),
+							commentId = urlPieces[ 4 ];
+						showCount( commentId, response[ path ].found );
+					}
+				}
+			},
+			error: function() {}
+		};
+
+		request( batchRequest );
+	}
+
+	function request( options ) {
```

Comment:
I told something similar to Marko, that `fetchCounts()` could be replaced by its code, and in turn `request()` replaced inside it. I didn't do it back then out of readability, but I do agree that this could be all consolidated in `getCommentLikeCounts()` since there's no other usage for these functions.